contracts/staking/contracts/src/sys/MixinParams.sol

```diff
@@ -231,6 +234,22 @@ contract MixinParams is
         );
     }
 
+    /// @dev Asserts that a minimum pool stake value is valid.
+    /// @param minimumStake Desired minimum pool stake.
+    function _assertValidMinimumPoolStake(
+        uint256 minimumStake
+    )
+        private
+        pure
+    {
+        if (minimumStake == 0) {
```

Comment:
Do we actually want this assertion? The difference between 0 and 1 seems pretty arbitrary here. I'd be fine removing this.

Reply:
The reward calculations explode if we allow a pool with stake 0 to get in the mix. Actually, this check should be `< 2` since delegator rewards are weighed less. I can special case these pools, but letting a pool with 0 stake collect rewards seems wrong to me?

Reply:
Sounds good. Let's move this assertion into `StakingProxy._assertValidStorageParams` now that #2166 is merged.